containers/CreateGridForm/components/AdminForm.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Box,
+  Button,
+  Container,
+  FormControl,
+  FormErrorMessage,
+  FormHelperText,
+  Heading,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  Stack,
+} from '@chakra-ui/react';
+import { useFormContext } from 'react-hook-form';
+import { FieldValues } from '..';
```

Comment:
because this is a type, do this 
```suggestion
import type { FieldValues } from '..';
```